installer/templates/phx_live/templates/page/home.html.leex

```diff
@@ -0,0 +1,48 @@
+<%%= if @live_view_action in [:deps] do %>
+  <%%= live_modal @socket, DemoWeb.HomeLive.DepsCheck,
+    id: "deps",
+    redirect_path: Routes.home_path(@socket, :index) %>
+<%% end %>
```

Comment:
I think you can re-use the original index.html.leex as long as you wrap this first line in a `<% if live do %>`. Less duplication this way. WDYT?